apis/core/v1alpha1/groupversion_info.go

```diff
@@ -2,8 +2,8 @@
 // SPDX-License-Identifier: Apache-2.0
 
 // Package v1alpha1 contains API Schema definitions for the core v1alpha1 API group
-//+kubebuilder:object:generate=true
-//+groupName=core.tanzu.vmware.com
+// +kubebuilder:object:generate=true
```

Comment:
Nit: sometimes in kubebuilder markers there is a space after the `//` comment tags and other times there aren't. I think we should just choose a format and stick with it. I think it's more common not to have that space, so I say remove these spaces, but this is a nit, also, so your choice.